src/serac/physics/tests/solid_multi_material.cpp

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) 2019-2024, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/solid_mechanics.hpp"
+
+#include "axom/slic/core/SimpleLogger.hpp"
```

Comment:
```suggestion
```
This isn't needed due to using the `serac::initiliaze` call which uses the Serac SLIC logger.

Reply:
Done